chaosk8s/node/actions.py

```diff
@@ -338,8 +338,8 @@ def drain_nodes(
 
         logger.debug(f"Found {len(eviction_candidates)} pods to evict")
         for pod in eviction_candidates:
-            eviction = client.V1beta1Eviction()
-
+            # eviction = client.V1beta1Eviction()
```

Comment:
You may remove that line rather than commenting it :)

Reply:
Sure